pkg/driver/controller.go

```diff
@@ -892,7 +902,7 @@ func (cs *controller) GetCapacity(
 		// See https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1472-storage-capacity-tracking#available-capacity-vs-maximum-volume-size &
 		// https://github.com/container-storage-interface/spec/issues/432 for more details
 		for _, zpool := range zfsNode.Pools {
-			if zpool.Name != poolParam {
+			if zpool.Name != poolParamPool {
```

Comment:
Can we rename poolParamPool to poolname?

Reply:
Of course. Will do that shortly.